cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/TaskWorkerTwillRunnable.java

```diff
@@ -154,9 +154,7 @@ public void destroy() {
   }
 
   private void doInitialize(TwillContext context) throws Exception {
-    CConfiguration cConf = CConfiguration.create();
-    cConf.clear();
-    cConf.addResource(new File(getArgument("cConf")).toURI().toURL());
+    CConfiguration cConf = CConfiguration.create(new File(getArgument("cConf")).toURI().toURL());
```

Comment:
Isn't this will miss the `cdap-default.xml` from the classpath??

Reply:
![image](https://user-images.githubusercontent.com/6337120/122345493-c4934b80-cefc-11eb-8080-530ebac68826.png)

cdap-default settings are in the cConf.writeXML output. 

This comment seems to be incorrect 
![image](https://user-images.githubusercontent.com/6337120/122345596-e12f8380-cefc-11eb-958c-947f287ca0bc.png)